yt_dlp/extractor/redtube.py

```diff
@@ -12,7 +12,7 @@
 
 
 class RedTubeIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:(?:\w+\.)?redtube\.com/|embed\.redtube\.com/\?.*?\bid=)(?P<id>[0-9]+)'
+    _VALID_URL = r'https?://(?:(?:\w+\.)?redtube\.(?:com|com\.br)?/|embed\.redtube\.com/\?.*?\bid=)(?P<id>[0-9]+)'
```

Comment:
```suggestion
    _VALID_URL = r'https?://(?:(?:\w+\.)?redtube\.com(?:\.br)?/|embed\.redtube\.com/\?.*?\bid=)(?P<id>[0-9]+)'
```
should the same be done for the `embed` URL format too?

Reply:
hello, they use the same embed URL so no changes are needed
![image](https://github.com/yt-dlp/yt-dlp/assets/1846222/a66b6ce3-8dc7-4b93-83c6-89912bfc0f32)